recipes/cryptominisat/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "cryptominisat" %}
+{% set version = "5.0.1" %}
+{% set sha256 = "789d53c90ecbd5910f48e4e1bd835e374b268e0b3872a81fd5c6178a30a5e263" %}
+{% set author = "msoos" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/{{ author }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: False
```

Comment:
It looks like this could be built for windows according to the project's README, but you will need to add a `bld.bat` file. Otherwise, you should add `skip: True  # [win]`